File: src/config/development.ts
import { Config } from "./Config";

const config: Config = {
  resqApiBaseUrl: "https://stroke.dev.qualityregistry.org/api/rest/hpfdc/",
  keycloakAuthUrl: "https://auth.dev.qualityregistry.org/realms/stroke/protocol/openid-connect/auth",
  oauthClientId: "doc-marker",
  uploadServerBaseUrl: "https://quest.ms.mff.cuni.cz/resq-doc-marker/backend/",
  uploadServerActInDevelopment: true,
  resqRecordUrl: (id) => `https://stroke.dev.qualityregistry.org/data-collection/record/${id}/form`,
};

export default config;

File: backend/app/config/development.py
from .Config import Config
from pathlib import Path

development_config = Config(
    resq_api_base_url="https://stroke.dev.qualityregistry.org/api/rest/hpfdc/",
    uploaded_files_storage_path=Path("storage", "development"),
    keycloak_token_url="https://auth.dev.qualityregistry.org/realms/stroke/protocol/openid-connect/token",
    oauth_client_id="doc-marker",
    oauth_client_password=None,
)
